java/mtsj/core/src/main/java/com/devonfw/application/mtsj/general/common/base/JWTAuthenticationFilter.java

```diff
@@ -18,11 +18,8 @@
  */
 public class JWTAuthenticationFilter extends GenericFilterBean {
 
-
-	/**
-	 * {@inheritDoc}
-	 */
-	@Override
+  @SuppressWarnings("javadoc")
```

Comment:
why do we need @SuppressWarnings("javadoc") in an overridden method? Seems as you Eclipse is just misconfigured. I thought with devon-ide we would not have such issues anymore but seems I am wrong. Can you check your eclipse settings?